Makefile

```diff
@@ -177,6 +177,18 @@ golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
 $(GOLANGCI_LINT): $(LOCALBIN)
 	$(call go-install-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint,${GOLANGCI_LINT_VERSION})
 
+.PHONY: cluster-up
```

Comment:
Is better to have the .PHONY next to the target, so if you remove rename the target you don't forget since its next to it, I will keep it like this, also this is how Makefile is generated by kubebuilder scaffolding.